backend/modules/ffmpeg/MediaContentApiHandler.js

```diff
@@ -69,7 +69,10 @@ class SubtitleApiHandler extends RequestHandler {
     const response = { subtitles };
 
     let { streams } = this.item.attributes;
-    if (!streams) streams = await FFProbe.getInfo(this.filePath).streams;
+    if (!streams) {
+      const probe = await FFProbe.getInfo(this.filePath);
+      ({ streams } = probe);
```

Comment:
the ()'s aren't needed here.
so that makes: `{ streams } = probe;`
please fix that and I'll merge it